tfgcv/testdata/full_compute_instance.tf

```diff
@@ -0,0 +1,121 @@
+resource "google_compute_instance" "full_list_default_1" {
+  # Required arguments
+  name         = "test1"
+  machine_type = "n1-standard-1"
+  boot_disk {
+    auto_delete             = true
+    device_name             = "test-device_name"
+    disk_encryption_key_raw = "test-disk_encryption_key_raw"
+    initialize_params {
+      # TODO: panic in google.resolveImageImageExists if it is not a global image
+      image = "projects/debian-cloud/global/images/debian-9"
+      size  = 42
+      type  = "pd-standard"
+    }
+    # TODO: generate new test case.
+    # Got: "boot_disk.0.kms_key_self_link": conflicts with boot_disk.0.disk_encryption_key_raw
+    # kms_key_self_link = "test-kms_key_self_link"
+    # TODO: report a bug.
+    # Got: An argument named "mode" is not expected here.
+    # mode = "READ_ONLY"
+    source = "test-source"
+  }
+  network_interface {
+    access_config {
+      nat_ip = "test-nat_ip"
```

Comment:
I think this should be a real IP? Or, even better, omit it entirely (as I think that's the best scenario to test/verify works correctly).

Reply:
updated with valid IP.